### PR TITLE
fix grammar in main readme

### PR DESCRIPTION
**Pull request by Dgarc359**
**Please describe the changes this PR makes and why it should be merged:**
Simple grammar fix for guards section of main readme

## Package
- discordx

### CODE REVIEW

File: packages/discordx/README.md [hunk 1]
```diff
@@ -369,7 +369,7 @@ class Example {
 
 # ⚔️ [Guards](https://discord-ts.js.org/docs/decorators/general/guard)
 
-We implemented a guard system thats work pretty like the [Koa](https://koajs.com/) middleware system
+We implemented a guard system that functions like the [Koa](https://koajs.com/) middleware system
```
Comment (oceanroleplay) [inline packages/discordx/README.md:372]:
```suggestion
We implemented a guard system that functions same as [Koa](https://koajs.com/) middleware system
```

Comment (oceanroleplay) [reply packages/discordx/README.md:372]: @VictoriqueMoe

Comment (oceanroleplay) [reply packages/discordx/README.md:372]: @AndyClausen @VictoriqueMoe

Comment (AndyClausen) [reply packages/discordx/README.md:372]: "like the" is more correct

**Review by AndyClausen — APPROVED** [review]

File: packages/discordx/README.md (hunk 1 above)
Comment (oceanroleplay) [reply packages/discordx/README.md:372]: ok

Comment (oceanroleplay) [reply packages/discordx/README.md:372]: okay, thanks

**Pull request merged**